src/main/java/emissary/place/VersionPlace.java

```diff
@@ -0,0 +1,78 @@
+package emissary.place;
+
+import emissary.core.IBaseDataObject;
+import emissary.core.ResourceException;
+import emissary.util.Version;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+public class VersionPlace extends ServiceProviderPlace {
+
+    private static final String EMISSARY_VERSION = "EMISSARY_VERSION";
+    private boolean includeDate;
+    private final Version version = new Version();
+    private String formattedVersion;
+
+    /**
+     * Create the place from the specified config file or resource
+     *
+     * @param configInfo the config file or resource to use
+     * @param dir the name of the controlling directory to register with
+     * @param placeLoc string name of this place
+     */
+    public VersionPlace(String configInfo, String dir, String placeLoc) throws IOException {
+        super(configInfo, dir, placeLoc);
+        configurePlace();
+    }
+
+    /**
+     * Create the place from the specified config stream data
+     *
+     * @param configInfo the config file or resource to use
+     * @param dir the name of the controlling directory to register with
+     * @param placeLoc string name of this place
+     */
+    public VersionPlace(InputStream configInfo, String dir, String placeLoc) throws IOException {
+        super(configInfo, dir, placeLoc);
+        configurePlace();
+    }
+
+    /**
+     * Create the place from the specified config stream data
+     *
+     * @param configInfo the config file or resource to use
+     */
+    public VersionPlace(InputStream configInfo) throws IOException {
+        super(configInfo);
+        configurePlace();
+    }
+
+    /**
+     * Create with the default configuration
+     */
+    public VersionPlace() throws IOException {
+        super();
+        configurePlace();
+    }
+
+    private void configurePlace() {
+        includeDate = configG.findBooleanEntry("INCLUDE_DATE", true);
+        formattedVersion = getEmissaryVersion();
```

Comment:
Realizing that this sounds a bit crazy and we should probably fix/simplify this. 
Do we want to use the version that is added to the git properties file during build time? 
https://github.com/NationalSecurityAgency/emissary/blob/main/src/main/java/emissary/util/GitRepositoryState.java#L154

This is used at startup to print the version.

Reply:
Didn't even realize this was a thing in the project haha. I think it may be smart right now to use the Version object to get this information, but in the future if we want to add commit hash/branch info, just use the GitRepoState class. Others may have differing opinions though

Reply:
yep agree